google_bigquery_writer/writer.py

```diff
@@ -121,12 +121,18 @@ def write_table(
             job_config.skip_leading_rows = 1
             job_config.allow_quoted_newlines = True
 
-            job = self.bigquery_client.load_table_from_file(
-                readable,
-                table_reference,
-                job_config=job_config
-            )
-
+            try:
+                job = self.bigquery_client.load_table_from_file(
+                    readable,
+                    table_reference,
+                    job_config=job_config
+                )
+            except bq_exceptions.BadRequest as err:
+                message = 'Cannot load table from file %s: %s' % (
+                    table_reference,
```

Comment:
bacha, `table_reference` je typ `bigquery.TableReference`, takže tohle by nemuselo projít? navrhuju 

```
            message = 'Loading data into table %s.%s failed: %s' % (
                dataset_name,
                table_definition['dbName'],
                str(err)
            )
```
(ukradeno z https://github.com/keboola/google-bigquery-writer/blob/9df07aa8423c0707951511ceb76efa5aefbfd81f/google_bigquery_writer/writer.py#L168)

Reply:
jj máš pravdu - opravil jsem to

Reply:
Háška `Cannot load table from file %s` vyznívá tak, že by ten parametr měl být název souboru a ne tabulky. Držel bych se tý chybový hlášky, co jsem linkoval, zbytečný vymýšlet novou.